docs/source/checks/vision/train_test_validation/plot_similar_image_leakage.py

```diff
@@ -1,130 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
I think the consideration was that we're not going to fix it now, and that it doesn't work well enough, but @Nadav_Barak please correct me here.

Reply:
yes exactly, currently its already out of the suite cause is not working well.
Making it work well under the new refactor would require a lot of effort we were not willing to make.
Making it work as bad as before would also require effort and some thinking since it uses indices for display - IMO not worth the investment

Reply:
Got it, thanks. Your last sentence as the one making it a clear cut to remove currently... (and to revisit when we're back to vision)